fastapi-sample2/main.py

```diff
@@ -0,0 +1,107 @@
+from fastapi import FastAPI,Depends,HTTPException
+
+from database_config import SessionLocal
+import crud_departments
+import crud_users
+from sqlalchemy.orm import Session
+import validation
+from typing import List
+
+app = FastAPI()
+
+# DB接続
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+#departments
+
+# get
+@app.get("/departments",response_model=List[validation.Departments])
+async def get_departments(db:Session=Depends(get_db)):
+ 
+    """
+        crud_departments.pyの関数でsqlを実行する
+        http://localhost:8000/departments
+    """
+    return crud_departments.get_departments(db=db)
+
+
+
+@app.get("/departments/{id}",response_model=validation.Departments)
+async def get_department(id:int,db:Session=Depends(get_db)):
+    
+    """
+        crud_departments.pyの関数でsqlを実行する
+        パスパラメーターで指定したidを使用してレコードを指定する
+        http://localhost:8000/departments/any
+    """
+    try:
+        result = crud_departments.get_department(id=id,db=db)
+        if result == None:
+            raise HTTPException(status_code=404, detail=f"{id}:Negative values are not accepted")
```

Comment:
```suggestion
            raise HTTPException(status_code=422, detail=f"{id}:Negative values are not accepted")
```
`code=404`を49行目で使用しておりますので、こちらのエラーコードは`422` もしくは `400`がよいかもしれません！

Reply:
レビューありがとうございます！
修正後もう一度リクエストいたします。
宜しくお願いします

Reply:
よろしくお願いします🙇

Reply:
再度レビューリクエストいたしました。
envで読み込むのもやってみました。
よろしくお願いします。